packages/jest-config/src/resolveConfigPath.ts

```diff
@@ -108,7 +108,7 @@ const resolveConfigPathByTraversing = (
     throw new ValidationError(...makeMultipleConfigsErrorMessage(configFiles));
   }
 
-  if (configFiles.length > 0 || packageJson) {
```

Comment:
packageJson checked earlier

Reply:
Then we don't need the `??` in the body either

Reply:
> Then we don't need the `??` in the body either

Yes, thanks, corrected